Microsoft/PCI_Compliance_for_Windows_JSON/README.md

```diff
@@ -0,0 +1,18 @@
+# PCI Compliance for Windows JSON
+
+This is an updated version of the PCI Compliance for Windows app. It is updated to work with Windows Event Logs in the JSON formatted syntax. It is also utilizing the new Sumo Logic dashboard framework. 
+
+## Setup
+
+### Setup Collection
+
+This app only requires a standard Windows Event source(s) configured for the JSON-formatted event logs.
```

Comment:
Recommend adding a link to our help docs to set up the Windows Events source with JSON format.